filebeat/prospector/prospector.go

```diff
@@ -21,6 +21,7 @@ var (
 )
 
 type Prospector struct {
+	harvesterCounter uint64         // Must be 8-byte aligned. Ensured if first field in struct
```

Comment:
I'm fine putting it under Bugfixes. 

Do you have some ideas on how to add a test? Would be great to have one.